docs/standard/microservices-architecture/container-docker-introduction/docker-defined.md

```diff
@@ -23,13 +23,13 @@ You can develop on Windows, Linux, or macOS. The development computer runs a Doc
 
 On Windows or macOS, [Docker Community Edition (CE)](https://www.docker.com/community-edition) hosts containers in a development environment and provides additional developer tools. [Docker Enterprise Edition (EE)](https://www.docker.com/enterprise-edition) is used by IT teams who build, ship, and run large business-critical applications. ~Both products install the necessary VM (the Docker host) to host the containers.~ 
 
-[Windows Containers](https://msdn.microsoft.com/en-us/virtualization/windowscontainers/about/about_overview) work with two types of runtimes:
+[Windows Containers](https://msdn.microsoft.com/virtualization/windowscontainers/about/about_overview) work with two types of runtimes:
```

Comment:
change this to https://docs.microsoft.com/virtualization/windowscontainers/about/index